CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ All notable changes to this project will be documented in this file, in reverse
 
 ### Changed
 
-- Nothing.
+-  update twig version and use namespaced twig classes #51.
```

Comment:
Please use the same format as is used elsewhere in the changelog.

Reply:
resolved.